manifests/init.pp

```diff
@@ -239,15 +238,16 @@
   }
   else {
     exec { 'Auto-generate JWK Key store file':
-      command  => "keytool -genkeypair -alias JWK -keyalg RSA -noprompt -dname 'CN=${server_base_url}, OU=ID, O=IRIDA, L=IRIDA,S=IRIDA,\
-       C=CA' -keystore ${jwk_key_store_path} -validity 3650 -storepass ${jwk_key_store_password} -keypass ${jwk_key_store_password}\
+      command     => "keytool -genkeypair -alias JWK -keyalg RSA -noprompt -dname \
+       'CN=${server_base_url}, OU=ID, O=IRIDA, L=IRIDA,S=IRIDA,\
+       C=CA' -keystore ${jwk_key_store_path} -validity 3650 -storepass \$PASS -keypass \$PASS \
```

Comment:
I think `\$PASS` should be wrapped in quotes so that it doesn't cause an issue if a special character is present, i.e. `!`